mlrun/feature_store/steps.py

```diff
@@ -46,10 +48,17 @@ def do(self, event):
         This method defines the do method of this class according to the first event type.
```

Comment:
why "first" event? it's being done on over every event, right? but in fact it would have been good if it will be done based on first event type - instead on each event.
please fix __init__() comment to include spark

Reply:
We call this method just on the first incoming event and when we overwrite this method... you can see it [here](https://github.com/mlrun/mlrun/pull/2777/files#diff-f76f1b5f9fc6cb5a415c5622b27bef40ae757bc96bf2cf447c5a95e87c2de982R56).